media-gfx/nifskope/nifskope-2.0.dev9.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="Tool for opening and editing NIF files"
+DESCRIPTION="NifSkope is a tool for opening and editing the NetImmerse file format (NIF). \
+NIF is used by video games such as Morrowind, Oblivion, Skyrim, Fallout 3/NV/4/76, Starfield, \
+Civilization IV, and more."
+HOMEPAGE="http://www.niftools.org/"
+COPYRIGHT="2016-2023 NifTools project"
+LICENSE="NIFSKOPE"
+REVISION="1"
+SOURCE_URI="https://github.com/hexabits/nifskope/archive/refs/tags/v2.0.dev9.tar.gz"
+CHECKSUM_SHA256="b40baca5fc3b11292cd284b5ef0d07dbe40c3bab8067c05b7f25bdb6141fe26c"
+SOURCE_URI_2="https://github.com/g-truc/gli/archive/refs/tags/0.8.2.0.tar.gz"
+CHECKSUM_SHA256_2="9e7024c2df77c011eff4f66667c1834620c70b7902cd50f32ab48edd49fe0139"
+SOURCE_URI_3="https://github.com/qhull/qhull/archive/refs/tags/v8.1-alpha1.tar.gz"
+CHECKSUM_SHA256_3="09e5e4c5b2b8a9e617a46876fef5a3d33e70aa1d08a163ff05d37701327c3be7"
+SOURCE_URI_4="https://github.com/niftools/nifxml/archive/refs/tags/v0.9.0.tar.gz"
+CHECKSUM_SHA256_4="984c247115bc49f90ded69b4c8feb9a62a0365ad53f5890312302b6a05bb4ed7"
+ADDITIONAL_FILES="nifskope.rdef.in"
+
+PATCHES="nifskope-2.0.dev9.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+    settings/nifskope/kfm.xml keep-old
+    settings/nifskope/nif.xml keep-old	
+    "
+	
+PROVIDES="
+    nifskope$secondaryArchSuffix = $portVersion
+    app:NifSkope$secondaryArchSuffix = $portVersion
+    app:nif.xml$secondaryArchSuffix = $portVersion
+    app:kfm.xml$secondaryArchSuffix = $portVersion
+    "
```

Comment:
only app:NifSkope should be in here.
I just noticed you used spaced in the start of the lines (no wonder it looked weird in the web interface) :)
We don't use spaces, use 1 tab at the start of each line (where needed)